### PR TITLE
(fix: #670): Copyright year is outdated it should be updated from `2023` to `2024`

### PR DESCRIPTION
**Pull request by subhamkumarr**
- use "Fixes" only when the pull request completely satisfies the issue

### Before Change:
WhatsApp Image 2024-01-24 at 12 56 55_b94d7145

### After changes:
Screenshot (1036)

#### changed in About section also
Screenshot (1037)

#### ✅️ By submitting this PR, I have verified the following
- [X] Checked to see if a similar PR has already been opened 🤔️
- [X] Reviewed the contributing guidelines 🔍️
- [X] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [X] Tried Squashing the commits into one

### CODE REVIEW

**Comment by subhamkumarr** [comment]
@tanmoysrt sir, i raised this 2days ago could you please review this?

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Interactive-Book",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```
Comment (tanmoysrt) [inline package-lock.json:6]:
```suggestion
```

Delete this file

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (subhamkumarr) [reply package-lock.json]: @tanmoysrt  Done the changes Sir.

Comment (subhamkumarr) [reply package-lock.json]: @tanmoysrt Done the changes sir.

**Comment by subhamkumarr** [comment]
@tanmoysrt  i made the changes.

**Comment by subhamkumarr** [comment]
@tanmoysrt @tachyons Could you please review this PR

**Review by tachyons — APPROVED** [review]

**Pull request merged**